specification/keyvault/resource-manager/readme.azureresourceschema.md

```diff
@@ -25,9 +25,7 @@ output-folder: $(azureresourceschema-folder)/schemas
 # all the input files in this apiVersion
 input-file:
   - Microsoft.KeyVault/preview/2020-04-01-preview/managedHsm.json
-  - Microsoft.KeyVault/preview/2020-04-01-preview/keyvault.json
```

Comment:
Why were these removed? With Key Vault and Managed HSM sharing RPs and data-plane swagger, versions must be in lock step or your forcing a bad customer UX where they have to use one api-version or another based on what they are connecting to. The idea was to make the experience transparent with the same APIs. That must include version.

/cc @johanste

Reply:
The actual KeyVault RP does not support these API versions. I'm all for adding them back in when they do, but it feels misleading to suggest them as options until then. We're having issues for this raised in https://github.com/Azure/bicep.

Reply:
Discussed this on an internal email chain - I'll leave this PR as purely additive and we can decide what to do with the unsupported resource types separately.